velox/type/Filter.cpp

```diff
@@ -145,15 +145,23 @@ bool BigintValuesUsingBitmask::testInt64Range(
   return !(min > max_ || max < min_);
 }
 
+template <typename T>
 BigintValuesUsingHashTable::BigintValuesUsingHashTable(
```

Comment:
May be rename class to `IntegerHashTableFilter`. Since we support int128 as well. Also suffixing with `Filter` makes it clear that it is a filter built using hash table.

Reply:
Thanks for the suggestion, renamed to ```Int128ValuesUsingHashTable``` to retain consistency with the current filter naming convention.